ui/app/src/contexts/AppRolesContext.ts

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+
+export const AppRolesContext = React.createContext({
+    roles: [] as Array<string>,
+    setAppRoles: (roles: Array<string>) => { }
+  });
```

Comment:
`nit` new line at end of file and also `});` should be flush with `export` (no space beforehand)